resources/assets/icons/moon-underline.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1 14.6h18m-5.6-5.9c-3 0-5.5-2.4-5.6-5.4 0-.6.1-1.3.2-1.9-2.9 1-4.5 4.1-3.5 7 1 2.9 4.1 4.5 7 3.5 1.7-.6 3-1.8 3.5-3.5-.5.2-1 .3-1.6.3z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
move `fill="none"` to the paths that needs it instead of the top-level svg element